pom.xml

```diff
@@ -600,6 +590,110 @@
 				<opensrp.database.profile>postgres</opensrp.database.profile>
 			</properties>
 		</profile>
+		<profile>
+			<id>FileSystem</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+				<property>
+					<name>opensrp.file.manager.type</name>
+					<value>FileSystem</value>
+				</property>
+			</activation>
+			<properties>
+				<opensrp.file.manager.profile>FileSystem</opensrp.file.manager.profile>
+			</properties>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-compiler-plugin</artifactId>
+						<configuration>
+							<excludes>
+								<exclude>**/OSSMultimediaFileManager.*</exclude>
+								<exclude>**/OSSClientBuilder.*</exclude>
+								<exclude>**/S3MultimediaFileManager.*</exclude>
+							</excludes>
+							<testExcludes>
+								<testExclude>**/OSSMultimediaFileManagerTest.*</testExclude>
+								<testExclude>**/S3MultimediaFileManagerTest.*</testExclude>
+							</testExcludes>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+		<profile>
+			<id>AWS_S3</id>
+			<activation>
+				<property>
+					<name>opensrp.file.manager.type</name>
+					<value>AWS_S3</value>
+				</property>
+			</activation>
+			<properties>
+				<opensrp.file.manager.profile>AWS_S3</opensrp.file.manager.profile>
+			</properties>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-compiler-plugin</artifactId>
+						<configuration>
+							<excludes>
+								<exclude>**/OSSMultimediaFileManager.*</exclude>
+								<exclude>**/OSSClientBuilder.*</exclude>
+							</excludes>
+							<testExcludes>
+								<testExclude>**/OSSMultimediaFileManagerTest.*</testExclude>
+							</testExcludes>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
+			<dependencies>
+				<!-- https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk -->
+				<dependency>
+					<groupId>com.amazonaws</groupId>
+					<artifactId>aws-java-sdk</artifactId>
+					<version>1.11.659</version>
+				</dependency>
+			</dependencies>
+		</profile>
+		<profile>
+			<id>AliCloud_OSS</id>
+			<activation>
+				<property>
+					<name>opensrp.file.manager.type</name>
+					<value>AliCloud_OSS</value>
+				</property>
+			</activation>
+			<properties>
+				<opensrp.file.manager.profile>AliCloud_OSS</opensrp.file.manager.profile>
+			</properties>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-compiler-plugin</artifactId>
+						<configuration>
+							<excludes>
+								<exclude>**/S3MultimediaFileManager.*</exclude>
+							</excludes>
+							<testExcludes>
+								<testExclude>**/S3MultimediaFileManagerTest.*</testExclude>
+							</testExcludes>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
```

Comment:
Maybe you you just import the dependency in test scope to allow to run all the tests